exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/text_ai/txaie_preprocessing.ipynb

```diff
@@ -0,0 +1,3564 @@
+{
```

Comment:
<p>Maybe add an explanation that each NLP extractor has its own configuration parameters, but the <code>Defaults</code> is a helper object allowing us to set these parameters once an apply these settings to all extractors. A heads up : this name is likely to change.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>